concordium-std/src/test_infrastructure/trie.rs

```diff
@@ -102,15 +74,179 @@ impl StateTrie {
     }
 }
 
-#[derive(Debug)]
+impl ContractStateApi for StateTrie {
```

Comment:
I don't think this makes sense to implement for StateTrie, as it cannot provide a complete host API.

Reply:
I agree. `ContractStateApi` trait should be removed.